docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -96,9 +98,121 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 WantedBy=multi-user.target default.target
 ```
 
+Creating systemd unit file for a pod with `--new` flag
+
+As its seen the generated unit files with `--new` flag has ExecStartPre and ExecStopPost service actions. ExecStartPre makes sure that each time the service is started it deletes any old reference left and then spins up a new instance. ExecStopPost removes the relevant instances after services are stopped.
+
+```
+$ sudo podman generate systemd --new --files --name bb310a0780ae
+# container-busy_moser.service
+# autogenerated by Podman 1.8.3-dev
+# Fri Apr  3 09:40:47 EDT 2020
+
+[Unit]
+Description=Podman container-busy_moser.service
+Documentation=man:podman-generate-systemd(1)
+Wants=network.target
+After=network-online.target
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+Restart=on-failure
+ExecStartPre=/usr/bin/rm -f %t/%n-pid %t/%n-cid
+ExecStart=/usr/local/bin/podman run --conmon-pidfile %t/%n-pid --cidfile %t/%n-cid --cgroups=no-conmon -d -dit alpine
+ExecStop=/usr/local/bin/podman stop --ignore --cidfile %t/%n-cid -t 10
+ExecStopPost=/usr/local/bin/podman rm --ignore -f --cidfile %t/%n-cid
+PIDFile=%t/%n-pid
+KillMode=none
+Type=forking
+
+[Install]
+WantedBy=multi-user.target default.target
+```
+
+### Installation of generated systemd unit file|s.
+
+Podman generated unit files include an `[Install]` section, which carries installation information for the unit. It is used by the enable and disable commands of systemctl(1) during installation.
+
+Once you have generated the systemd unit file, you can copy the generated systemd file to ```/usr/lib/systemd/system``` for installing as a root user and to ```$HOME/.config/systemd/user ``` for installing it as a non-root user. Now, Enable the copied one or more units using `systemctl enable`.
+
+Note: Coping a unit to ```/usr/lib/systemd/system``` and enabling it marks the unit to be automatically started at boot.
+
+
+```
+# Copy and enable all the generated systemd files.
+
+$ sudo cp pod-systemd-pod.service container-great_payne.service /usr/lib/systemd/system
+$ systemctl enable pod-systemd-po.service
+Created symlink /etc/systemd/system/multi-user.target.wants/pod-systemd-po.service → /usr/lib/systemd/system/pod-systemd-po.service.
+Created symlink /etc/systemd/system/default.target.wants/pod-systemd-po.service → /usr/lib/systemd/system/pod-systemd-po.service.
+$ systemctl is-enabled pod-systemd-po.service
+enabled
+```
+To run the user services placed in ~/.config/systemd/user/ on first login of that user, execute
```

Comment:
```suggestion
To run the user services placed in `$HOME/.config/systemd/user/` on first login of that user, execute
```

Otherwise, only `/.config/...` is displayed in the man page.

Reply:
Fixed.